src/lighting.cpp

```diff
@@ -103,6 +106,50 @@ void Lighting::updateLightMatrices(struct boundingbox* sceneAabb)
     bx::mtxInverse(lightingState.lightInvViewProjTexMatrix, lightingState.lightViewProjTexMatrix);
 }
 
+void Lighting::ff7_load_ibl()
+{
+	struct game_mode* mode = getmode_cached();
+	static uint32_t prev_mode = -1;
+	static char filename[64]{ 0 };
+	static char specularFullpath[MAX_PATH];
+	static char diffuseFullpath[MAX_PATH];
+	static WORD last_field_id = 0, last_battle_id = 0;
+
+	switch (mode->driver_mode)
+	{
+	case MODE_BATTLE:
+		if (mode->driver_mode != prev_mode || last_battle_id != ff7_externals.modules_global_object->battle_id)
+		{
+			last_battle_id = ff7_externals.modules_global_object->battle_id;
+
+			sprintf(filename, "bat_%d", last_battle_id);
+			sprintf(specularFullpath, "%s/%s/%s%s", basedir, external_ibl_path.c_str(), filename, "_s.dds");
+			sprintf(diffuseFullpath, "%s/%s/%s%s", basedir, external_ibl_path.c_str(), filename, "_d.dds");
```

Comment:
Same goes for those two. Sorry I missed them.

Reply:
Fixed this as well.